sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1407,15 +1406,15 @@ static int create_sdw_dailink(struct snd_soc_card *card,
 		if (cpu_dai_id[i] != ffs(adr_link_next->mask) - 1)
 			continue;
 
-		ret = create_codec_dai_name(dev, adr_link_next, codecs, codec_idx,
+		ret = create_codec_dai_name(dev, adr_link_next, codecs, codec_index,
 					    codec_conf, codec_count, codec_conf_index,
 					    adr_index, dai_index);
 		if (ret < 0)
 			return ret;
 
 		/* check next link to create codec dai in the processed group */
 		i++;
-		codec_idx += adr_link_next->num_adr;
+		codec_index += adr_link_next->num_adr;
```

Comment:
does this really make the code better/easier to read?

codec_idx is a counter, while codec_index is assigned line 1421. It looks like a different concept.

@RanderWang @bardliao please chime in on this one.

Reply:
One is the index into the codec_info array, the other is the index into the codecs dai_link_comps array. I am happy to keep them separate if we prefer but renaming them so they arn't just called the same thing. (idx=index)